shopinvader_payment/services/cart.py

```diff
@@ -3,21 +3,15 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 
-from odoo.addons.component.core import Component
+from odoo.addons.component.core import AbstractComponent, Component
 
 
-class CartService(Component):
-
-    _inherit = "shopinvader.cart.service"
+class AbstractPayableSaleService(AbstractComponent):
```

Comment:
This abstract should be declared in a dedicated file.... abstract_payable_sale_service.py